### PR TITLE
Fixes Bug 690332: [Feedback] Visual Studio for Mac IDE Themes Look Wrong

### PR DESCRIPTION
**Pull request by mkrueger**
Mac do not work for XAML

https://devdiv.visualstudio.com/DevDiv/_workitems/edit/690332

> [Fixes VSTS #690332](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/690332)
> [Related:  VSTS #681833](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/681833)

### CODE REVIEW

File: main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor.Highlighting/Formats/OldFormat.cs [hunk 1]
```diff
@@ -1401,8 +1456,12 @@ public static ColorScheme Import (string fileName, Stream stream)
 							}
 						}
 					}
-					if (!found && !importedAmbientColors.Contains (vsc.Name))
-						Console.WriteLine (vsc.Name + " not imported!");
+
+					if (!found && !importedAmbientColors.Contains (vsc.Name)) {
+						var isOptional = textColors.TryGetValue (vsc.Name, out var pd) && pd.Attribute.IsOptional;
+						if (!isOptional)
+							Console.WriteLine (vsc.Name + " not imported!");
```
Comment (Therzok) [inline main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor.Highlighting/Formats/OldFormat.cs]: Logging service, maybe?

**Comment by slluis** [comment]
@monojenkins backport release-7.7

**Comment by slluis** [comment]
@mkrueger tests are failing

**Comment by mkrueger** [comment]
@monojenkins backport release-7.7

**Pull request merged**